templates/rhpam75-prod-immutable-kieserver-amq.yaml

```diff
@@ -165,12 +165,12 @@ parameters:
   value: mykeystorepass
   required: false
 - displayName: KIE Server Bypass Auth User
-  description: KIE server bypass auth user (Sets the org.kie.server.bypass.auth.user system property)
+  description: Allows the KIE server to bypass the authenticated user for task-related operations, for example, queries. (Sets the org.kie.server.bypass.auth.user system property)
   name: KIE_SERVER_BYPASS_AUTH_USER
   value: 'false'
   required: false
 - displayName: KIE Server Container Deployment
-  description: 'KIE Server Container deployment configuration in format: containerId=groupId:artifactId:version|c2=g2:a2:v2'
+  description: 'KIE Server Container deployment configuration with optional alias. Format: containerId=groupId:artifactId:version|c2(alias2)=g2:a2:v2'
```

Comment:
Will let the review of this field to @errantepiphany.

Reply:
I got this change from @errantepiphany and he reviewed it already in the master branch - this is a cherry-pick

Reply:
ok, no problem.